lib/galaxy/managers/workflows.py

```diff
@@ -486,6 +489,10 @@ def build_invocations_query(
         ]
         return invocations, total_matches
 
+    def delete(self, stored_workflow, flush=True):
+        """Mark the given workflow deleted."""
+        super().delete(stored_workflow, flush=flush)
+
```

Comment:
This seems to just defer to `DeletableManagerMixin.delete`, can we skip this ?

Reply:
done in https://github.com/galaxyproject/galaxy/pull/14794/commits/0eed6c5374aed8a5ab115fe10169e1289e4b0e57